minibear/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -0,0 +1,248 @@
+import frc.robot.common.PortSpy;
+
+/**
+ * This class is where the bulk of the robot should be declared. Since
+ * Command-based is a
+ * "declarative" paradigm, very little robot logic should actually be handled in
+ * the {@link Robot}
+ * periodic methods (other than the scheduler calls). Instead, the structure of
+ * the robot (including
+ * subsystems, commands, and button mappings) should be declared here.
+ */
+public class RobotContainer {
+
+  private SubsystemCollection subsystems = new SubsystemCollection();
+  private final AutonomousChooser autonomousChooser;
+
+  /**
+   * The container for the robot. Contains subsystems, OI devices, and commands.
+   */
+  public RobotContainer() {
+
+    // init the pdp watcher
+    this.initializePowerDistributionPanelWatcherSubsystem();
+
+    // init the various subsystems
+    this.initializeDrivetrainSubsystem();
+
+    // init the input system
+    this.initializeManualInputInterfaces();
+
+    // init wrist system
+    this.initializeWristSubsystem();
+
+    // arm and picker later
+    this.initializeIntakeSubsystem();
+
+    // calculate and update the current position of the robot
+    this.calculateAndUpdateRobotPosition();
+
+    // Configure the button bindings
+    System.out.println(">>>> Initializing button bindings.");
+    this.subsystems.getManualInputInterfaces().initializeButtonCommandBindings();
+    System.out.println(">>>> Finished initializing button bindings.");
+    new ZeroWristEncoder(this.subsystems.getWristSubsystem());
+    System.out.println(">>>> Finished Initing Wrist");
+
+    this.initializeDebugDashboard();
+    this.autonomousChooser = new AutonomousChooser(subsystems);
+    // TODO refactor the commands in ManualInputInterfaces:
+    // bindBasicDriveToPointButtonsToDriverXboxController and
+    // bindDriveTrajectoryButtonsToDriverXboxController
+    // to instead be commands on the shuffleboard like this:
+    // SmartDashboard.putData("Example Command", exampleCommand);
+
+    // Command to drive the chassis for zeroing the swerve modules.
+    SmartDashboard.putData("Drive Forward Robot Centric",
+      new DriveTimeCommand(this.subsystems.getDriveTrainSubsystem(), 
+      new ChassisSpeeds(0.6, 0.0, 0.0), 3.0));  
+
+    SmartDashboard.putData("Drive Forward with rotation",
+    new DriveTimeCommand(this.subsystems.getDriveTrainSubsystem(), 
+    new ChassisSpeeds(0.6, 0.0, 0.2), 3.0));
+    SmartDashboard.putData("Print NavX State",
+        new InstantCommand(this.subsystems.getDriveTrainSubsystem()::printState));
+
+    SmartDashboard.putData("Zero Wrist", new ZeroWristEncoder(this.subsystems.getWristSubsystem()));
+
+
+  }
+
+  /**
+   * Use this to pass the autonomous command to the main {@link Robot} class.
+   *
+   * @return the command to run in autonomous
+   */
+  public Command getAutonomousCommand() {
+    // An ExampleCommand will run in autonomous
+    return autonomousChooser.getCommand();
+  }
+
+  /**
+   * 
```

Comment:
I suggest we describe.  If nothing less we should do stuff like:

   /**
    * {@inheritdoc}
    */